demo/src/app/app.component.ts

```diff
@@ -117,8 +117,11 @@ export class AppComponent implements OnInit {
     charCounterCount: false,
     toolbarInline: true,
     events: {
-      'froalaEditor.initialized': function() {
-        console.log('initialized');
+      'froalaEditor.initialized': function(e, editor) {
+        editor.events.on('click', function(){
```

Comment:
Is this fix for #264? If so, then it is not OK as if we click on the middle of the editor the cursor would go at the end of the editor instead of at the beginning. We should be checking if the editor is empty by using the `core.isEmpty` method.